lib/modules/story_editor/widgets/actor_editor_dialog.dart

```diff
@@ -0,0 +1,96 @@
+import 'package:andax/models/actor.dart';
+import 'package:andax/models/translation_asset.dart';
+import 'package:andax/modules/story_editor/screens/story_editor.dart';
+import 'package:andax/shared/widgets/editor_dialog.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+void showActorEditorDialog(
+  BuildContext context,
```

Comment:
After further inspection, it turns out that the full context object is required anyways because of the `showEditorDialog` function call on line 28, which uses `showDialog` (from Flutter itself) which requires the context.